src/routes/home/components/CreateGameDialog.vue

```diff
@@ -119,10 +119,12 @@ export default {
           gameName: this.gameName,
           isRanked: this.isRanked,
         })
-        .then(() => {
+        .then((res) => {
+          const { gameId } = res;
           this.gameName = '';
           this.loading = false;
           this.show = false;
```

Comment:
I don't think these are necessary anymore since vue should destroy this component on pushing to the `lobby` view

Reply:
ok, you are right - deleted